internal/handlers/common/aggregations/expression.go

```diff
@@ -132,18 +133,17 @@ func NewExpression(expression string) (*Expression, error) {
 }
 
 // Evaluate gets the value at the path.
-// It returns `types.Null` if the path does not exists.
-func (e *Expression) Evaluate(doc *types.Document) any {
+// It returns error if the path does not exists.
+func (e *Expression) Evaluate(doc *types.Document) (any, error) {
 	path := e.path
 
 	if path.Len() == 1 {
 		val, err := doc.Get(path.String())
 		if err != nil {
-			// $group stage groups non-existent paths with `Null`
-			return types.Null
+			return nil, err
```

Comment:
We can create `ExpressionError` right?

Reply:
I assume this one is the same as https://github.com/FerretDB/FerretDB/pull/2789/files#r1227739197